### PR TITLE
[v1.2] Use 1.2 version in manifests and charts as default image version

### PR DESCRIPTION
**Pull request by zimnx**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def setup(sphinx):
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ['master', 'v0.3', 'v1.0', 'v1.1']
+BRANCHES = ['master', 'v0.3', 'v1.0', 'v1.1', 'v1.2']
```
Comment (tnozicka) [inline docs/source/conf.py:159]: isn't this only used from the `master` branch?

**Review by tnozicka — COMMENTED** [review]
- pls squash the 2 `make update` commits into 1
- pls link the PR that you are backporting

**Comment by tnozicka** [comment]
I've found a prerequisite PR, that's why this has 2 `make update` commits I guess. Pls mark it as WIP/draft next time and link the prerequisite PR in the description.

File: docs/source/conf.py (hunk 1 above)
Comment (zimnx) [reply docs/source/conf.py]: yep, this change is meant for master. sorry

**Comment by tnozicka** [comment]
Would `[v1.2] Pin manifests to v1.2` be a more descriptive title then `[v1.2] release: prepare for v1.2.0-rc.0` as this isn't related to the specific tag but the branch.

**Review by tnozicka — DISMISSED** [review]
/lgtm

**Pull request merged**